internal/hardware/validator.go

```diff
@@ -576,3 +575,44 @@ func compileDiskReasonTemplate(template string, wildcards ...interface{}) *regex
 	}
 	return tmp
 }
+
+// preciseHumanizeBytes provides more precise disk size formatting for error messages
```

Comment:
```suggestion
// preciseHumanizeBytes provides more precise disk size formatting for error messages
```
I think comments should be adapted, more precise to what? The previous code will be deleted, so there is nothing to compare. Comments should describe what does the function.